debian/ubuntu-advantage-tools.postinst

```diff
@@ -265,6 +265,24 @@ mark_reboot_cmds_as_needed() {
     add_notice "$msg_name"
 }
 
+check_wrong_fips_metapackage_on_cloud() {
+      fips_metapkg="ubuntu-fips"
```

Comment:
Note whitespace offset seems to be 6 spaces indent here instead of 4-based like the rest of the file